Header header

int32 fixmode
int32 nsv
float64 latitude
float64 longitude
float64 altitude
float32 forwardv
float32 north
float32 east
float32 down
float32 headingAngle
float32 roll
float32 pitch
float32 yaw
float32 rollRate
float32 pitchRate
float32 yawRate